wordle-web/src/App.vue

```diff
@@ -1,28 +1,15 @@
 <template>
   <header>
-    <div class="wrapper">
-      <nav>
-        <RouterLink to="/">Home</RouterLink> | <RouterLink to="/wordle">Wordle</RouterLink> |
-        <RouterLink to="/about">About</RouterLink>
-      </nav>
-      <v-btn @click="setInverseTheme"> Inverse Theme </v-btn>
-      <v-btn @click="setDarkTheme"> Dark Theme </v-btn>
-    </div>
+    
   </header>
-
+  <nav-bar></nav-bar>
```

Comment:
This nav bar ends up covering some of the game boxes. Would add something to make sure the bar doesn't cover any important content.